data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/amazon_es.mapping.yml

```diff
@@ -0,0 +1,10 @@
+pluginName: opensearch
+mappedAttributes:
+  hosts: hosts
+  user: username
```

Comment:
Data Prepper will not use the `user` or `password` fields when coming from the `amazon_es` plugin.

Also, Data Prepper should translate the `aws_access_key_id` and `aws_secret_access_key` properties.

See the plugin documentation: https://github.com/awslabs/logstash-output-amazon_es

Reply:
I'm still awaiting the final mapping from @jianghancn 
Till then, I'll take a look at the documentation and update this.

Reply:
@sshivanii , It should just be a matter of mapping from the link above to the targets defined here: 
https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/opensearch

I believe we should be able to wrap this up without @jianghancn.

Reply:
Yes I can take a look at the doc and map it myself too. I just wanted @jianghancn to confirm the final mapping since he is the Sink author. But I'll send him this PR and he can confirm the mapping then.